contracts/safeguard/sentinel/Guard.sol

```diff
@@ -29,28 +29,31 @@ abstract contract Guard is Ownable {
         _setRelaxThreshold(guards.length);
     }
 
-    function updateGuardState(GuardState _state) external {
-        GuardState current = guardStates[msg.sender];
-        require(current != GuardState.None, "caller is not guard");
-        require(current != _state, "same with current state");
-        guardStates[msg.sender] = _state;
-        if (_state == GuardState.Guarded) {
-            numRelaxedGuards--;
-        } else if (_state == GuardState.Relaxed) {
-            numRelaxedGuards++;
-        } else {
-            revert("invalid update");
-        }
+    // change GuardState of msg.sender from relaxed to guarded
+    function guard() external {
+        require(guardStates[msg.sender] == GuardState.Relaxed, "invalid caller");
+        guardStates[msg.sender] = GuardState.Guarded;
+        numRelaxedGuards--;
+        _updateRelaxed();
+        emit GuardUpdated(msg.sender, GuardState.Guarded);
+    }
+
+    // change GuardState of msg.sender from guarded to relaxed
+    function relax() external {
+        require(guardStates[msg.sender] == GuardState.Guarded, "invalid caller");
+        guardStates[msg.sender] = GuardState.Relaxed;
+        numRelaxedGuards++;
         _updateRelaxed();
-        emit GuardUpdated(msg.sender, _state);
+        emit GuardUpdated(msg.sender, GuardState.Relaxed);
     }
 
-    function addGuards(address[] calldata _accounts, bool _incrementThreshold) external onlyOwner {
+    function addGuards(address[] calldata _accounts, uint256 _thresholdIncrement) external onlyOwner {
```

Comment:
```solidity    
    function addGuards(address[] calldata _accounts, uint256 _newRelaxThreshold) external onlyOwner {
        for (uint256 i = 0; i < _accounts.length; i++) {
            _addGuard(_accounts[i]);
        }
        setRelaxThreshold(_newRelaxThreshold);
    }

    function removeGuards(address[] calldata _accounts, uint256 _newRelaxThreshold) external onlyOwner {
        for (uint256 i = 0; i < _accounts.length; i++) {
            _removeGuard(_accounts[i]);
        }
        setRelaxThreshold(_newRelaxThreshold);
    }
 ```
 Cleaner?

Reply:
merged into a single cleaner `updateGuards` function to minimize owner operation overhead